pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -287,7 +287,8 @@ in rec {
           name = attrs.name or "${attrs.pname}-${attrs.version}";
 
           # If the packager hasn't specified `outputsToInstall`, choose a default,
-          # which is the name of `p.bin or p.out or p`;
+          # which is the name of `p.bin or p.out or p` along with `p.man` when
+          # present;
           # if he has specified it, it will be overridden below in `// meta`.
```

Comment:
Since you're editing this, can we remove the unnecessarily gendered pronoun here?